FrontendApartmentReservation/src/i18n.js

```diff
@@ -35,7 +41,13 @@ i18n.use(LanguageDetector).init({
         "Password": "Slaptažodis",
         "Office" : "Ofisas",
         "SignUpError": "Klaida registruojantis:",
-        "PasswordError": "Slaptažodis turi turėti vieną didžiąją vieną ir mažąją raides, specialų simbolį ir būti 5-15 simbolių ilgio"
+        "PasswordError": "Slaptažodis turi turėti vieną didžiąją vieną ir mažąją raides, specialų simbolį ir būti 5-15 simbolių ilgio",
```

Comment:
Change it to "bent jau 8 simbolių ilgio", since regex has changed to match current password standard